Include/cpython/unicodeobject.h

```diff
@@ -46,9 +46,11 @@
     Py_UNICODE_ISDIGIT(ch) || \
     Py_UNICODE_ISNUMERIC(ch))
 
+/* Py_DEPRECATED(3.3) */
 #define Py_UNICODE_COPY(target, source, length) \
```

Comment:
Is it possible to convert some macros into static inline functions to be able to put Py_DEPRECATED() on them?